message Value
{
	required string id = 1;
	optional int64 indegree = 2;
	repeated string parents = 3;
	repeated string children = 4;
	repeated string dataNameList = 5;
	repeated int64 dataSize = 6;
	optional int64 allDataSize = 7;
	optional int64 tasklength = 8;
	
	optional int64 numTaskFin = 9;
	optional int64 numWorkSteal = 10;
	optional int64 numWorkStealFail = 11;
	optional int32 numTaskWait = 12;
	optional int32 numTaskReady = 13;
	optional int32 numCoreAvilable = 14;
	optional int32 numAllCore = 15;
	
	optional int64 outputsize = 16;
	
	optional int64 submitTime = 17;
}